target/linux/ramips/dts/mt7628an_tplink_tl-wr902ac-v4.dts

```diff
@@ -0,0 +1,99 @@
+#include "mt7628an_tplink_8m.dtsi"
+
+/ {
+	compatible = "tplink,tl-wr902ac-v4", "mediatek,mt7628an-soc";
+	model = "TP-Link TL-WR902AC v4";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		sw1 {
+			label = "sw1";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+
+		sw2 {
+			label = "sw2";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_1>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 41 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		lan {
+			label = "green:lan";
```

Comment:
Perhaps use `color` and `function` properties instead of `label`.

Reply:
> Perhaps use `color` and `function` properties instead of `label`.

Where can I read about it? Or you want simple textual replacement:
                       color = <LED_COLOR_ID_GREEN>;
                       function = LED_FUNCTION_LAN;
(but still ...)

Reply:
Yes, from these the actual label will be derived.
More info: https://elixir.bootlin.com/linux/v5.15.137/source/Documentation/devicetree/bindings/leds/common.yaml

Reply:
@raenye next time to make it easier for people, instead of making them guess what you mean... [just use the suggestion feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#:~:text=to%20suggest%20a%20specific%20change%20to%20the%20line%20or%20lines)

Reply:
Thanks, wasn't aware of it.